app/build.gradle.kts

```diff
@@ -4,10 +4,9 @@ import config.ApplicationConfig.androidMinSdk
 import config.ApplicationConfig.androidTargetSdk
 
 plugins {
-    id(config.Plugins.appPlugin)
-    kotlin(config.Plugins.androidPlugin)
-    kotlin(config.Plugins.kapt)
-    id(config.Plugins.hiltPlugin)
+    alias(libs.plugins.appPlugin)
+    alias(libs.plugins.androidPlugin)
+    alias(libs.plugins.kaptPlugin)
```

Comment:
what about Hilt?

Reply:
I wonder what is the best choice for kmm 🤔             
Koin? Kodein-DI? I had it as a separate task in mind (let me create [a card for this](https://trello.com/c/XjfgtNR2/13-choose-and-integrate-di-for-kmm) on the dashboard).